src/widgets/buttons.ts

```diff
@@ -27,6 +27,16 @@ export type StatusAreaContext = {
   div?: HTMLElement
   dom?: HTMLDocument
 }
+export enum ButtonType {
+  Primary = '#7c4dff',
+  Secondary = '#01C9EA',
+  Cancel = '',
+  Continue = ''
+}
+export type ButtonWidgetOptions = {
+  buttonType?: ButtonType,
```

Comment:
I would've like to make this mandatory, but so that I didn't break code I made them optional and then defaulted the style to Primary, filled.  Any objections?